src/components/VMediaInfo/VCopyLicense.vue

```diff
@@ -34,91 +34,26 @@
+        <div
           v-if="tab === 'rich'"
-          id="attribution-rich"
-          path="media-details.reuse.credit.text"
-          tag="p"
-        >
-          <template #title
-            ><VLink :href="media.foreign_landing_url">{{
-              media.title
-            }}</VLink></template
-          >
-          <template #creator>
-            <i18n
-              v-if="media.creator"
-              path="media-details.reuse.credit.creator-text"
-              tag="span"
-            >
-              <template #creator-name>
-                <VLink v-if="media.creator_url" :href="media.creator_url">{{
-                  media.creator
-                }}</VLink>
-                <span v-else>{{ media.creator }}</span>
-              </template>
-            </i18n>
-          </template>
-          <template #marked-licensed>
-            {{
-              isPDM
-                ? $t('media-details.reuse.credit.marked')
-                : $t('media-details.reuse.credit.licensed')
-            }}
-          </template>
-          <template #license>
-            <VLink class="uppercase" :href="licenseUrl">{{
-              fullLicenseName
-            }}</VLink
-            >{{ period }}
-          </template>
-        </i18n>
+          v-html="getAttributionMarkup({ includeIcons: false })"
+        />
+        <!-- eslint-enable vue/no-v-html -->
+
         <p
           v-if="tab === 'html'"
           id="attribution-html"
           class="font-mono break-all"
           dir="ltr"
         >
-          {{ attributionHtml }}
+          {{ getAttributionMarkup() }}
         </p>
-        <i18n
-          v-if="tab === 'plain'"
-          id="attribution-plain"
-          path="media-details.reuse.credit.text"
-          tag="p"
-        >
-          <template #title>{{ media.title }}</template>
-          <template #creator>
-            <i18n
-              v-if="media.creator"
-              path="media-details.reuse.credit.creator-text"
-            >
-              <template #creator-name>{{ media.creator }}</template>
-            </i18n>
-          </template>
-          <template #marked-licensed>
-            {{
-              isPDM
-                ? $t('media-details.reuse.credit.marked')
-                : $t('media-details.reuse.credit.licensed')
-            }}
-          </template>
-          <template #license> {{ fullLicenseName }}</template>
-          <template #view-legal>
-            <i18n path="media-details.reuse.credit.view-legal-text">
-              <template #terms-copy>
-                {{
-                  isPDM
-                    ? $t('media-details.reuse.credit.terms-text')
-                    : $t('media-details.reuse.credit.copy-text')
-                }}
-              </template>
-              <template v-if="licenseUrl" #URL>
-                {{ licenseUrl }}
-              </template>
-            </i18n>
-          </template>
-        </i18n>
+
+        <div v-if="tab === 'plain'">
```

Comment:
This is missing the `id` attribute used by the copy button for grabbing the copyable text. The "rich text" one is missing it as well. This breaks the copy button for this PR.

When you fix this, would you mind adding an e2e test for this behavior, just to check that copying the text works for each of the tabs?

Reply:
Good catch, I'll fix it right away and add the tests.